packages/contracts-bedrock/.storage-layout

```diff
@@ -53,15 +51,15 @@
 +---------------+-------------------------------------------------+------+--------+-------+----------------------------------------------------+
 | Name          | Type                                            | Slot | Offset | Bytes | Contract                                           |
 +==============================================================================================================================================+
-| messenger     | contract CrossDomainMessenger                   | 0    | 0      | 20    | contracts/L1/L1StandardBridge.sol:L1StandardBridge |
+| _initialized  | uint8                                           | 0    | 0      | 1     | contracts/L1/L1StandardBridge.sol:L1StandardBridge |
 |---------------+-------------------------------------------------+------+--------+-------+----------------------------------------------------|
-| otherBridge   | contract StandardBridge                         | 1    | 0      | 20    | contracts/L1/L1StandardBridge.sol:L1StandardBridge |
+| _initializing | bool                                            | 0    | 1      | 1     | contracts/L1/L1StandardBridge.sol:L1StandardBridge |
```

Comment:
Need to figure out how to make this diff work. Worst case we can move all of the init into the child contracts. A bit less clean but at least it maintains the storage layout. We can work it out towards the end.

Reply:
I this we should do state surgery to fix it, we will need to write the same sort of functionality tests with or without state surgery used to fix it and its a very simple surgery

Reply:
Can't use surgery on the L1 side though

Reply:
ah good point

Reply:
The surgery on L1 would be more complex, could do a chugsplash style upgrade where we upgrade to state setter then upgrade to implementation right?